src/analyze.js

```diff
@@ -6,7 +6,7 @@ export function analyzeTrackFeatures(trackFeatures) {
 		const mean = getMean(rawData);
 		const deviation = getStandardDeviation(rawData);
 		analysis[dimension] = {mean, deviation};
-		console.log(`tracks have an average ${dimension} of ${mean.toFixed(3)} with a deviation of ${deviation.toFixed(3)}`);
+		console.log(`tracks have an average ${dimension} of ${roundPlaces(mean, 2)} with a deviation of ${roundPlaces(deviation, 2)}`);
```

Comment:
not related to this PR, but need to remember to use `roundPlaces` instead of `toFixed` in the bar rendering code, where the user is likely to see values that don't get truncated properly